File: java/Baju.java
class Baju extends Aksesoris {
    private String untuk;
    private String size;
    private String merk;

    public Baju(Integer ID, String nama, String kategori, Integer harga, String jenis, String bahan, String warna, String untuk, String size, String merk) {
        super(ID, nama, kategori, harga, jenis, bahan, warna);
        this.untuk = untuk;
        this.size = size;
        this.merk = merk;
    }

    public String getUntuk() {
        return untuk;
    }

    public void setUntuk(String untuk) {
        this.untuk = untuk;
    }

    public String getSize() {
        return size;
    }

    public void setSize(String size) {
        this.size = size;
    }

    public String getMerk() {
        return merk;
    }

    public void setMerk(String merk) {
        this.merk = merk;
    }
}

File: python/Baju.py
from Aksesoris import Aksesoris

class Baju(Aksesoris):
    def __init__(self, ID=None, nama=None, kategori=None, harga=None, jenis=None, 
                 bahan=None, warna=None, untuk=None, size=None, merk=None):
        # Panggil parent class constructor
        super().__init__(ID, nama, kategori, harga, jenis, bahan, warna)
        # Atribut Baru
        self._untuk = untuk
        self._size = size
        self._merk = merk

    def getUntuk(self):
        """Fungsi untuk mendapatkan target pengguna baju"""
        return self._untuk

    def getSize(self):
        """Fungsi untuk mendapatkan ukuran baju"""
        return self._size

    def getMerk(self):
        """Fungsi untuk mendapatkan merk baju"""
        return self._merk

    def setUntuk(self, untuk):
        """Fungsi untuk menetapkan target pengguna baju"""
        self._untuk = untuk

    def setSize(self, size):
        """Fungsi untuk menetapkan ukuran baju"""
        self._size = size

    def setMerk(self, merk):
        """Fungsi untuk menetapkan merk baju"""
        self._merk = merk